spec/hallmonitor/outputters/dogstatsd_outputter_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'spec_helper'
+require 'hallmonitor/outputters/dogstatsd_outputter'
+
+module Hallmonitor
+  module Outputters
+    RSpec.describe DogstatsdOutputter do
+      let(:dogstatsd_client) { instance_double(Datadog::Statsd) }
+      let(:prefix) { 'test' }
+      let(:outputter) { described_class.new(prefix) }
+
+      before do
+        allow(dogstatsd_client).to receive(:namespace=)
+        allow(Datadog::Statsd).to receive(:new).and_return(dogstatsd_client)
+      end
+
+      it 'can be instantiated' do
+        expect { outputter }.to_not raise_error
+      end
+
+      context '#process' do
+        let(:event_name) { 'foo.bar.baz' }
+        let(:event_tags) { {tags: [{foo:"bar"}]} }
+        let(:event_tags_expected) {{tags:["tags:[{:foo=>\"bar\"}]"]}}
```

Comment:
I expected it too but it did come back as  :foo=>"bar". I assumed it might just be a ruby thing

```
  1) Hallmonitor::Outputters::DogstatsdOutputter#process with an event sends the event to statsd
     Failure/Error: @statsd.count(event_name, event.count, tags: process_tags(event.tags))
     
       #<InstanceDouble(Datadog::Statsd) (anonymous)> received :count with unexpected arguments
         expected: ("foo.bar.baz", 1, {:tags=>["tags:[{foo:\"bar\"}]"]})
              got: ("foo.bar.baz", 1, {:tags=>["tags:[{:foo=>\"bar\"}]"]})
```